File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L1/09/Level1.ts
import { Scanner } from 'readline-sync';

function reverse(number: number): void {
  while (number !== 0) {
    const remainder = number % 10;
    console.log(remainder);
    number = Math.floor(number / 10);
  }
  console.log();
}

function main(): void {
  console.log('Enter an integer: ');
  const input = new Scanner();
  const number = input.nextInt();
  reverse(number);
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L1/09/Level1.kt
package IR

/**
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level1 {
    @JvmStatic
    fun main(args: Array<String>) {
        println("Enter an integer: ")
        val input = java.util.Scanner(System.`in`)
        val number = input.nextInt()
        reverse(number)
    }

    //fungsi membalik
    private fun reverse(number: Int) {
        var num = number
        while (num != 0) {
            val remainder = num % 10
            print(remainder)
            num /= 10
        }
        println()
    }
}

